From 8cdffa56dd47681db35fd1a718c6ae14a62fa0c3 Mon Sep 17 00:00:00 2001
From: Matheus Sampaio Queiroga <srherobrine20@gmail.com>
Date: Tue, 21 Jul 2026 19:03:18 +0000
Subject: [PATCH 136/153] dt-bindings: net: omci: add optional dying-gasp alarm

Add an opt-in property for transmitting a best-effort ONU-G dying-gasp
alarm over OMCI in addition to the GPON PLOAM dying-gasp message.

Signed-off-by: Matheus Sampaio Queiroga <srherobrine20@gmail.com>
---
 Documentation/devicetree/bindings/net/omci.yaml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/net/omci.yaml b/Documentation/devicetree/bindings/net/omci.yaml
index 569b0ed12645..49e372f669b2 100644
--- a/Documentation/devicetree/bindings/net/omci.yaml
+++ b/Documentation/devicetree/bindings/net/omci.yaml
@@ -84,5 +84,11 @@ properties:
         minItems: 1
         maxItems: 20
 
+  omci-dying-gasp:
+    type: boolean
+    description:
+      Enable a best-effort ONU-G dying-gasp alarm notification over OMCI in
+      addition to the GPON PLOAM dying-gasp message. Disabled by default.
+
 additionalProperties: true
 ...
-- 
2.55.0

